pkg/providers/ovirt/provider.go

```diff
@@ -155,6 +155,14 @@ func (o *OvirtProvider) Init(secret *corev1.Secret, instance *v2vv1alpha1.Virtua
 	if len(o.ovirtSecretDataMap["caCert"]) == 0 {
 		return fmt.Errorf("oVirt secret caCert cannot be empty")
 	}
+	client, err := o.getClient()
```

Comment:
In the past we wanted to have lazy client to limit number of the connections if not needed. It would be great if we could create `TestConnection` method otherwise we would test the connection in every loop we make.